lib/webinspector/transformer/webinspector-decoder.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable promise/prefer-await-to-callbacks */
+import Stream from 'stream';
+import { parse } from '../../util/plist';
```

Comment:
I find the name of this function too generic

Reply:
I can change it when I move it to `appium-support`